ci-operator/step-registry/openshift-qe/node-density/openshift-qe-node-density-ref.yaml

```diff
@@ -62,4 +62,4 @@ ref:
       Default is true, which means clean up the pod/resource that kube-burner ocp created, you can set it to false to keep the resouce
   documentation: >-
     This step runs the perfscale node-density workload in the deployed cluster
-  timeout: 2h30m
+  timeout: 4h
```

Comment:
? Wow node-density taking longer than 2.5h ?

Reply:
yeah. I just would prefer to avoid silly timeouts so more breathing room the better. yeah, in a separate PR would be good -- juggling too many changes :D